.husky/pre-commit

```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-npx lint-staged
+# npx lint-staged # Why do we need this?  We have github actions for this and this breaks my IDE.
```

Comment:
Was there ever a discussion about using husky in the first place?

We have never used it as not all clients work with husky - the most obvious being Visual studio code. If you can't rely on the hooks running the IMHO it's better to remove husky completely and rely on GitHub actions.

Reply:
To elaborate on what Kris said - it may looks like this step is just running a linter to *check* for formatting errors. In fact, it is running a linter to *fix* formatting errors, and then *check those fixes in*.

If you only had a GHA to check for linting issues, you'd be constantly annoyed by failing linting that you'd have to fix and re-publish. If you run linting regularly , it will frequently complain about unrelated files that aren't the ones you're trying to commit.

lint-staged fixes the easy problems in the background, keeping the code clean with minimal annoyance to the developer.

I would highly recommend keeping lint-staged in place.